### ISSUE TITLE
new sites

### ISSUE THREAD

**Issue by doktorjasper** [SerjSX/pSearch#11]
here have a few new sites for Cg, and plugins pls add new categorie for cg and developer tools

https://gfxfather.com/
https://softarchive.is
https://rendercrate.com/3D
https://news.productioncrate.com
https://gfxplugin.com/en/product/14451/download-flip-fluids
https://www.cgpersia.com
https://www.0daydown.com 
https://polyhaven.com
https://sharecg.com

**Comment by SerjSX** [comment]
Deal, I'll keep you updated!

**Comment by SerjSX** [comment]
I could not validate the sites you mentioned on piracy megathread / fmhy, if you can share me the link of the megathread you got those from. So far I found these that I can put under GFX category (not CG but check they may include similar things):
- https://softarchive.is/ 
- https://www.gfxtra31.com/
- https://gfxmountain.com/
- https://www.avxgfx.com/
- https://www.onlygfx.com/
- https://cgarchives.com/

Plus be more specific with developer tools! Mobile, computer...?

**Comment by doktorjasper** [comment]
these web sites has unreal engine game development plugins, and other softwares. and cg, gfx, 3d models for the game development . game models . unreal engine marketplace

contents,


________________________________
Gönderen: Serj ***@***.***>
Gönderildi: 25 Aralık 2024 Çarşamba 11:33
Kime: SerjSX/pSearch ***@***.***>
Bilgi: enes çelebi ***@***.***>; Author ***@***.***>
Konu: Re: [SerjSX/pSearch] ne sites (Issue #11)


I could not validate the sites you mentioned on piracy megathread / fmhy, if you can share me the link of the megathread you got those from. So far I found these that I can put under GFX category (not CG but check they may include similar things):

  *   https://softarchive.is/
  *   https://www.gfxtra31.com/
  *   https://gfxmountain.com/
  *   https://www.avxgfx.com/
  *   https://www.onlygfx.com/
  *   https://cgarchives.com/

Plus be more specific with developer tools! Mobile, computer...?

**Comment by SerjSX** [comment]
Okay I see. I'll look into it and update you here

**Comment by doktorjasper** [comment]
https://cgpersia.com/2024/05/unreal-engine-marketplace-bundle-1-may-2024-195423.html
https://cgpersia.com/?s=kitbash
https://softarchive.is/blogs/gfxnews/fluid_flux_for_unreal_engine.4506006.html


________________________________
Gönderen: Serj ***@***.***>
Gönderildi: 27 Aralık 2024 Cuma 16:01
Kime: SerjSX/pSearch ***@***.***>
Bilgi: enes çelebi ***@***.***>; Author ***@***.***>
Konu: Re: [SerjSX/pSearch] ne sites (Issue #11)


Okay I see. I'll look into it and update you here

**Comment by SerjSX** [comment]
Okk I see what content you're looking after. I will check on freemediaheckyeah and piracy megathread for such links, I'll save them as a list, then I'll check and add them. 

Meanwhile, you can use "all" in the first site input box to search for the software or content you're looking for in all of the websites.

I'll let you know here once done

**Comment by SerjSX** [comment]
Current list to add:
Category: assets

- [ ] https://softarchive.is/
- [x] https://www.gfxtra31.com/
- [x] https://www.onlygfx.com/
- [x] https://cgarchives.com/
- [x] https://cgpersia.com/
- [x] https://unityassetcollection.com/
- [x] https://unityassets4free.com/

**Comment by SerjSX** [comment]
I will name the category "assets" when adding sites regarding this topic

**Comment by SerjSX** [comment]
Hello. I wanted to update you on this. 
I added the sites, but due to code issues, I have to manually make a new release to add the Assets shortcut. I am re-organizing the code in order to make it more readable both for me and for others. When I wrote the program, I had just finished Python. Now I also took a Python course at my university, and I am going to apply many of the things I learned to make the code easily readable.

As you can see I checked the sites that I added to the JSON, but I did not push it yet because category searching doesn't work with the Assets, so you have to manually drag down the dropdown list or type the site name to search which is inconvenient (I understand). I am going to do it such a way that this part is automated, so in the future such issues do not happen, and then I will make a new release.

I won't close this issue since I will update you here as well. Thanks for understanding

**Comment by doktorjasper** [comment]
thank you <3

**Comment by SerjSX** [comment]
Almost done, so far:

- Made shortcuts on main page to show 3 per each row, and it's scrollable 
- Fixed the results page scrolling so it works by mouse, loads faster too cause I removed unnecessary elements.
- Moved to OOP: GUI, websites functionalities, and scraping/site functionalities are in separate files for a better organization.
- Fixed minor code issues and removed unnecessary ones.
- More comments. 
- Fixed the loading bar so it would be more accurate to the size of the websites searching in, it was not correct before. 

I will look into other minor things too especially related to the types being automated fully.

**Comment by doktorjasper** [comment]
thank you so much. did you commited it ? as an exe as an exe

**Comment by SerjSX** [comment]
> thank you so much. did you commited it ? as an exe as an exe 

Wlc! Nope I did not commit anything yet, I'll do so once everything is done in less than a week or so.

**Comment by doktorjasper** [comment]
okey also i have another link  may you want to look. category is music development software- DAW softwares andplugins . https://4download.net/

**Comment by doktorjasper** [comment]
where u from. What are your future plans?

**Comment by SerjSX** [comment]
> where u from. What are your future plans?

I'll check the site you sent. I'm from Middle East! So far other than what I already did in the past couple of days, not much. I did all that I had to do, the code needed reorganization and some cleaning (when I re-read the code I noticed some flaws and fixed)

I work on it on my free time cause I have university studies, so if I automate the types and the button icons then I will stop there and make a new release before my semester starts.

I occasionally check the health of the sites and update the websites, but I don't do a lot of releases as the JSON file is automatically downloaded already whenever needed. Wanted to make the program as ready and automated as possible so I wouldn't make a new release every time I add a new site.

**Comment by SerjSX** [comment]
After new years I will double check and commit! Been busy. Happy holidays and new year in advance :)

**Comment by SerjSX** [comment]
pushed. Preparing to do a new release

**Comment by SerjSX** [comment]
Done thank you, closing this now. You can check the new release! If you face any issue, open a new issue please for it. Thx
